### ISSUE TITLE
consolidate.js dependency is not up to date on npm, cannot render twing. Fix here.

### ISSUE THREAD

**Issue by digitaltoast** [forwardemail/email-templates#405]
Looks like consolidate.js hasn't been updated on npm for years, so lots of template engines are missing.
To get the latest version .... 
```
npm install tj/consolidate.js#master
cd node_modules/email-templates
npm install tj/consolidate.js#master
````

I *had* to find a different template engine, I thought it was just me who instantly hated everything about pug!
https://www.quora.com/Do-web-developers-use-tools-like-Jade-Pug-rather-than-code-in-raw-HTML/answer/Frank-M-Taylor-1
"_I hate Jade/Pug with the passion of a thousand burning suns_" - I know how he feels!

I love this email-templates package, but might I suggest *anything* but pug for newbies?
If they want the sado-masochistic coding equivalent of nails on a blackboard, then offer pug as an option, but for newbies at least offer something friendly :)

**Comment by niftylettuce** [comment]
I just merged a few PR's and bumped `consolidate` to v0.16.0 @ https://github.com/tj/consolidate.js/releases/tag/v0.16.0.

Pug is currently the best template language in existence in my opinion and that is why I support it across everything.  Steer clear of React, SPA's, and TypeScript.  My work with @ladjs is soon going to be instrumental in helping the community build apps with ease, and it is used with in projects like https://forwardemail.net (completely open source @forwardemail) and https://ourharvest.com.

**Comment by niftylettuce** [comment]
Also someone leaving a post on Quora, stating they have a certificate in programming and front-end development, and one from an online campus does not warrant it to be an expert opinion.  I'm releasing a new version of email-templates with this bumped in just a moment.

**Comment by shaunwarman** [comment]
Out of curiosity is there a template engine you would prefer as an option? @digitaltoast

**Comment by niftylettuce** [comment]
v7.1.1 released to npm, thank you https://github.com/forwardemail/email-templates/releases/tag/v7.1.1

**Comment by digitaltoast** [comment]
> I just merged a few PR's and bumped `consolidate` to v0.16.0 @ https://github.com/tj/consolidate.js/releases/tag/v0.16.0.

Wow - I just realised that you actually do ALL these projects! I feel bad for being critical now as you are ridiculously productive in open source; thank you so much for these releases.

But I still really struggle with Pug - so many syntactical tripwires and a steep learning curve. Whereas, with Twig (Twing), it's the same old standard html we all understand, with clearly defined `{{tags}}`. I really appreciate the work you've done to bring so many template engines on board which means whichever engine we prefer, we can use it. Thank you!
